### PR TITLE
Delegate clair setup instructions to original docs

### PR DESCRIPTION
**Pull request by jseppi**
Rather than have our own clair instructions instructions, which can get out of date and become incorrect, this PR replaces them with links to the official docs.

### CODE REVIEW

File: pages/how-federalist-works/cloud-gov-setup.md [hunk 1]
```diff
@@ -135,19 +96,13 @@ docker-compose up
...
-To actually scan the image we'll use the [Cliar CLI Tool](https://github.com/coreos/clair/tree/master/contrib/analyze-local-images).
+To actually scan the image we'll use the [Clair analyze-local-images CLI tool](https://github.com/coreos/analyze-local-images).
 
-First install the CLI tool:
+First install the CLI tool according to the instructions at https://github.com/coreos/analyze-local-images#install
 
-``` shell
-go get -u github.com/coreos/clair/contrib/analyze-local-images
-```
+Clair's CLI tool copies the image to a `tmp` directory where Clair picks it up. In order for this to work, you need to change the value of the `TMPDIR` variable to `/tmp`: `export TMPDIR=/tmp`
```
Comment (jseppi) [inline pages/how-federalist-works/cloud-gov-setup.md]: moved this closer to the actual running of the tool.

File: pages/how-federalist-works/cloud-gov-setup.md [hunk 2]
```diff
@@ -79,53 +79,14 @@ Before the image can be pushed to the registry, it will need to be scanned by [C
...
-Clair will look for a config file in `./clair_config/config`. We recommend using [Clair's example config](https://github.com/coreos/clair/blob/master/config.example.yaml):
-
-``` shell
-curl -L https://raw.githubusercontent.com/coreos/clair/v1.2.2/config.example.yaml -o ./config/config.yaml
-```
+We recommend starting Clair with [Docker Compose](https://docs.docker.com/compose/).
+See the instructions at https://github.com/coreos/clair#docker-compose for how to set this up.
 
-Once those parts are in place. Clair can be started with Docker Compose. It takes Clair a while to update it's vulnerability definitions. Give it a few hours or your scan results will come up empty.
+It takes Clair a while to update its vulnerability definitions. Give it a few hours or your scan results will come up empty.
```
Comment (jseppi) [inline pages/how-federalist-works/cloud-gov-setup.md]: It seems like updating only took a few minutes, not hours.

**Comment by wslack** [comment]
defering to @jmhooper

**Comment by jseppi** [comment]
There is the change in scan results process stuff (ie, uploading to Google Drive folder instead of updating POAM spreadsheet) that would be good to have your review on, @wslack.

**Comment by jmhooper** [comment]
Yeah, this is 👍  with me, but I agree it would be good to have @wslack sign off on the policy changes.

**Pull request merged**